packages/block-library/src/file/edit.js

```diff
@@ -196,17 +196,17 @@ function FileEdit( { attributes, setAttributes, noticeUI, noticeOperations } ) {
 			</BlockControls>
 			<div { ...blockProps }>
 				<div className={ 'wp-block-file__content-wrapper' }>
-					<div className="wp-block-file__textlink">
-						<RichText
-							tagName="div" // must be block-level or else cursor disappears
-							value={ fileName }
-							placeholder={ __( 'Write file name…' ) }
-							withoutInteractiveFormatting
-							onChange={ ( text ) =>
-								setAttributes( { fileName: text } )
-							}
-						/>
-					</div>
+					<RichText
+						style={ { display: 'inline-block' } }
```

Comment:
Is this preferred to using CSS?

Reply:
This one is particular, I think using CSS is favored but I preferred an inline style here because it's more explicit that this is needed for RichText to work. (It's not just a style, it's behavior in a way)